unit_test/tstAPIC.hpp

```diff
@@ -583,10 +583,13 @@ void collocatedTest()
         "fill_grid_vector", TEST_EXECSPACE(),
         grid_vector->layout()->indexSpace( Cajita::Own(), Cajita::Local() ),
         KOKKOS_LAMBDA( const int i, const int j, const int k, const int d ) {
-            gv_view( i, j, k, d ) = 0.0000000001 * ( d + 1 ) * pow( i, 5 ) -
-                                    0.0000000012 * pow( ( d + 1 ) + j * i, 3 ) +
-                                    0.0000000001 * pow( i * j * k, 2 ) +
-                                    pow( ( d + 1 ), 2 );
+            int ic = i - 2;
+            int jc = j - 2;
+            int kc = k - 2;
```

Comment:
after

Reply:
This change was to deal with the new boundary halo in Cajita. Hence the local grid was offset by the halo size, even if there is no periodic boundary.